### PR TITLE
adding member-operator-config

### PR DESCRIPTION
**Pull request by tinakurian**
Making the IdP name configurable via a ConfigMap. 

Related to: https://issues.redhat.com/browse/CRT-743
Related PRs: https://github.com/codeready-toolchain/host-operator/pull/255 
                      https://github.com/codeready-toolchain/member-operator/pull/182

### CODE REVIEW

**Review by alexeykazakov — COMMENTED** [review]
Well, I think we could keep the IdP name as "rhd" by default in our toolchain-infra scripts.
But if we really want to make it configurable even here then we should also use it in https://github.com/codeready-toolchain/toolchain-infra/blob/master/config/oauth/idp.yaml#L9

**Review by alexeykazakov — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -128,6 +128,13 @@ The url where the registration-service is running.
 export REGISTRATION_SERVICE_URL=registration-service-url
 ----
 
+=== IDENTITY_PROVIDER
+The identity provider. If IDENTITY_PROVIDER env var is not provided, it will default to "rhd"
+
+[source]
+----
+export IDENTITY_PROVIDER=identity-provider
+
```
Comment (MatousJobanek) [inline README.adoc:138]: you forgot to add the end of the code snippet: 
```suggestion
----

```

**Review by MatousJobanek — COMMENTED** [review]
@tinakurian please, fix the README

File: README.adoc (hunk 1 above)
Comment (tinakurian) [reply README.adoc:138]: good catch, updated here: 8d4a044c54f96b0c85cafe44961e868eaf523780

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**